docker/nginx/nginx.conf

```diff
@@ -0,0 +1,88 @@
+#ssl_certificate /etc/letsencrypt/live/myopenhab.org/fullchain.pem;
+#ssl_certificate_key /etc/letsencrypt/live/myopenhab.org/privkey.pem;
+worker_processes 4;
+
+events { worker_connections 1024; }
+
+http {
+
+upstream app_server {
+        server app-1:3000 fail_timeout=5s max_fails=5;
+}
+
+server {
+    listen 8080 default_server;
```

Comment:
the usual port would be 80, why is this hardcoded to 8080? If this is intentional, because another webserver should proxy to this one, I would argue, that proxying directly to ohcloud would be more efficient, right? But maybe I just miss something :)